### ISSUE TITLE
strict-export-declare-modifiers and export as namespace

### ISSUE THREAD

**Issue by scottescue** [Microsoft/dtslint#115]
I have an index.d.ts file similar to the following:
```TypeScript
export as namespace Foo;

export class Person {
    name: string;
}
```

dtslint fails the strict-export-declare-modifiers rule on the line: `export as namespace Foo;`. Is this behavior correct? If so, what is the correct way to both export for module loaders and provide a global namespace (UMD style)? dtsgen generates an example index.d.ts that follows this syntax, so if the dtslint behavior is correct it appears there's a disconnect between the two tools.

I apologize for asking the question in an issue, but I don't know of a better place to ask.

**Comment by scottescue** [comment]
Closing as this appears to have been only happening locally when testing a new DefinitelyTyped package. 

The unexpected error went away when I installed typescript@latest (2.9.0-dev.20180506) into DefinitelyTyped. I also created a new strict-export-declare-modifiers test in dtslint and could not duplicate the issue.

**Comment by thw0rted** [comment]
I've been working on a few DT packages myself and it seems like something breaks at every turn -- would you mind explaining in a little more detail what you did to fix it?

**Comment by andy-ms** [comment]
@thw0rted Have you tried `rm -r node_modules; rm package-lock.json; npm install`? If so, what's a simple file you're seeing the error on?

**Comment by thw0rted** [comment]
Yes, cleaning my node_modules fixed it -- I'm vaguely curious as to how exactly my install was corrupted but probably not enough to take the time to track it down.  I'll keep this in mind next time I see weird behavior cropping up when testing DT.  Thanks!
